docs/data/base/components/number-input/number-input.md

```diff
@@ -91,6 +91,82 @@ The following code snippet:
 />
 ```
 
+### Events
```

Comment:
Does this only apply to the component (so, not also the hook)? If so then this is a good place for this content; if it also applies to the hook, then it should go under the "Customization" header.

Reply:
Hmm, a small part of this also applies to the hook but that part is quite important so I think it may be better to duplicate it

Reply:
In that case, I think I would prefer to follow the pattern we've already established and just have one Events section under Customization that applies to both components and hooks. The Hook section has the text about how hooks can't use `slots` or `slotProps`, but we could clarify that again in the Events section if you think it's needed. So then text like "`onChange` can only be passed as a prop on the component—not through `slotProps`" could become "When working with the component, `onChange` can only be passed..."

Reply:
👌 ~ Updated and moved to Customization: https://deploy-preview-39407--material-ui.netlify.app/base-ui/react-number-input/#events